app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/view/OnTabReselectedDispatcher.kt

```diff
@@ -15,14 +16,13 @@ class OnTabReselectedListener(private val viewPager: ViewPager) : TabLayout.OnTa
     }
 
     override fun onTabReselected(tab: TabLayout.Tab) {
-        val adapter = viewPager.adapter as FragmentStatePagerAdapter
-        val item = adapter.getItem(tab.position)
-        if (item is TabLayoutItem) {
-            item.scrollToTop()
+        val item = viewPager.adapter?.instantiateItem(viewPager, tab.position)
+        if (item is OnTabReselectedListener) {
```

Comment:
`adapter.getItem(tab.position)` creating new Fragment each time. So crashed.
We using instantiateItem() isntead of getItem. instantiateItem() use the cache.